lib/services/local/sessiontracker_test.go

```diff
@@ -87,3 +89,52 @@ func TestSessionTrackerStorage(t *testing.T) {
 	require.Error(t, err)
 	require.Nil(t, session)
 }
+
+func TestSessionTrackerImplicitExpiry(t *testing.T) {
+	ctx := context.Background()
+	bk, err := memory.New(memory.Config{})
+	require.NoError(t, err)
+
+	id := uuid.New().String()
+	id2 := uuid.New().String()
+	srv, err := NewSessionTrackerService(bk)
+	require.NoError(t, err)
+
+	_, err = srv.CreateSessionTracker(ctx, &proto.CreateSessionTrackerRequest{
+		Namespace:   defaults.Namespace,
+		ID:          id,
+		Type:        types.KindSSHSession,
+		Hostname:    "hostname",
+		ClusterName: "cluster",
+		Login:       "root",
+		Initiator: &types.Participant{
+			ID:   uuid.New().String(),
+			User: "eve",
+			Mode: string(types.SessionPeerMode),
+		},
+		Expires: time.Now().UTC().Add(time.Second),
+	})
+	require.NoError(t, err)
+
+	_, err = srv.CreateSessionTracker(ctx, &proto.CreateSessionTrackerRequest{
```

Comment:
Reuse the initial request, so it's clearer what changes between invocations?

Reply:
Taken care of